utils/node_local_test.py

```diff
@@ -516,7 +516,7 @@ def start(self, clean=True):
             self._check_timing("start", start, max_start_time)
         print('Server started in {:.2f} seconds'.format(time.time() - start))
 
-    def stop(self, wf):
+    def stop(self, abort_on_warning=True):
```

Comment:
(pylint-inconsistent-return-statements) Either all return statements in a function should return an expression, or none of them should.